internal/cfg/config.go

```diff
@@ -0,0 +1,199 @@
+	}
+	fileBindingsStr := ""
+	for src, dest := range c.FilesBindings {
+		fileBindingsStr = fmt.Sprintf("%s\t\t%s -> %s\n", fileBindingsStr, src, dest)
+	}
+	configStr := fmt.Sprintln(
+		"\tRepositories:\n", repoNamesStr,
+		"\tFiles bindings:\n", fileBindingsStr,
+		"\tDry Run:", c.IsDryRun,
+		"\n\tIs GitHub token set?", (len(c.GithubToken) != 0),
+		"\n\tGithub host URL: ", c.GithubURL,
+		"\n\tCommit message: ", c.CommitMessage,
+		"\n\tFile sync branch regexp: ", c.FileSyncBranchRegexp,
+		"\n\tWorkspace: ", c.Workspace,
+		"\n\tFile Source Path: ", c.FileSourcePath,
+	)
+	fmt.Println(configStr)
+}
+
+func getRepositoryNames() ([]string, error) {
+	// get the raw list from env
+	repoNamesStr := os.Getenv("REPOSITORIES")
+	if repoNamesStr == "" {
+		return nil, fmt.Errorf("REPOSITORIES is empty but required")
+	}
+	// trim spaces
+	repoNamesStr = strings.TrimSpace(repoNamesStr)
+	// split by \n
+	repoNames := strings.Split(repoNamesStr, "\n")
+
+	for _, name := range repoNames {
+		if len(strings.Split(name, "/")) != 2 {
+			return nil, fmt.Errorf("invalid repo name: %s {OWNER}/{NAME} expected", name)
+		}
+	}
+	return repoNames, nil
+}
+
+func getFilesBindings() (map[string]string, error) {
+	// get the raw list from env
+	filesBindingsStr := os.Getenv("FILES_BINDINGS")
+	if filesBindingsStr == "" {
+		return nil, fmt.Errorf("FILES_BINDINGS is empty but required")
+	}
+	// trim spaces
+	filesBindingsStr = strings.TrimSpace(filesBindingsStr)
+	// split by \n
+	fileBindingsList := strings.Split(filesBindingsStr, "\n")
+
+	filesBindings := make(map[string]string, len(fileBindingsList))
+	// split each binding by `=` to build the binding map
+	for _, fileBindingStr := range fileBindingsList {
+		split := strings.Split(fileBindingStr, "=")
+		if len(split) != 2 {
+			return nil, fmt.Errorf("incorrect binding: %s", fileBindingStr)
+		}
+		filesBindings[split[0]] = split[1]
+	}
+
+	// error if no files bindings have been found
+	if len(filesBindings) == 0 {
+		return nil, fmt.Errorf("no valid files bindings found")
+	}
+	return filesBindings, nil
+}
+
+func getDryRun() (bool, error) {
+	isDryRunStr := os.Getenv("DRY_RUN")
+	// default is true
+	if isDryRunStr == "" {
+		log.Infof("DRY_RUN empty: set to default value")
+		return true, nil
+	}
+	return strconv.ParseBool(isDryRunStr)
+}
+
+func getGithubToken() (string, error) {
+	githubToken := os.Getenv("GITHUB_TOKEN")
+	if githubToken == "" {
+		return "", fmt.Errorf("GITHUB_TOKEN is empty but required")
+	}
+	return githubToken, nil
+}
+
+func getGithubURL() (string, error) {
+	githubURL := os.Getenv("GITHUB_URL")
+	if githubURL == "" {
+		return "", fmt.Errorf("GITHUB_URL is empty but required")
+	}
+	return githubURL, nil
+}
+
+func getPRTitle() (string, error) {
+	prTitle := os.Getenv("PR_TITLE")
+	if prTitle == "" {
+		return "", fmt.Errorf("PR_TITLE is empty but required")
+	}
+	// auto-truncate pr title - 80 characters maximum
+	if len(prTitle) > 100 {
+		prTitle = prTitle[:99]
+	}
```

Comment:
```suggestion
	// auto-truncate pr title - 80 characters maximum
	if len(prTitle) > 80 {
		prTitle = prTitle[:79]
	}
```
80 or 100 characters?